Week 1/Homework/calculator.rb

```diff
@@ -0,0 +1,23 @@
+if ARGV.length == 2
+	amount_of_money = ARGV[0].to_f
+	amount_of_months = ARGV[1].to_f
+	interest_rate = rand(5..20).to_f
+	puts "Interest Rate: " + interest_rate.to_i.to_s + "%"
+	[*0..amount_of_months.to_i].each do |i|
+		amount_of_dollars = amount_of_money.to_i
+		amount_of_cents = ((amount_of_money % 1) * 100).to_i
+		amount_of_dollars_string = amount_of_dollars.to_s
```

Comment:
Since you are only using this variable one, you are actually over abstracting in this case. This is not necessary. You can just write it into line 18. Also in ruby you don't have to type cast to concatenate variables.